shared/js/smart-screen/clock.js

```diff
@@ -55,6 +55,7 @@ function Clock() {
       this.timeoutID = window.setTimeout(function cl_setClockInterval() {
 
         if (self.timerID == null) {
+          refresh(new Date());
```

Comment:
I believe this patch works. How about make a patch the same as [1] ? Phone already had it.
[1] https://github.com/mozilla-b2g/gaia/commit/aaf7934bb4bd27f19f1d12434636542a2ab136d4